src/main/java/com/prokhor/mini_system/services/StudentsService.java

```
@@ -0,0 +1,141 @@
+
+@Getter
+public class StudentsService {
+    private final List<Student> students = new LinkedList<>();
+    private final List<Course> courses = new ArrayList<>();
+
+    private static final int SCHOLARSHIP_AMOUNT;
+    private static final int SCHOLARSHIP_GRADE;
+    private static final int PRESIDENT_SCHOLARSHIP_GRADE;
+    private static final int PRESIDENT_PERCENTAGE_INCREASE;
+
+    static {
+        try {
+            SCHOLARSHIP_AMOUNT = Integer.parseInt(PropertiesLoader.loadProperties().getProperty("scholarship_amount"));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    static {
+        try {
+            SCHOLARSHIP_GRADE = Integer.parseInt(PropertiesLoader.loadProperties().getProperty("scholarship_grade"));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    static {
+        try {
+            PRESIDENT_SCHOLARSHIP_GRADE = Integer.parseInt(PropertiesLoader.loadProperties().getProperty("president_scholarship_grade"));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    static {
+        try {
+            PRESIDENT_PERCENTAGE_INCREASE = Integer.parseInt(PropertiesLoader.loadProperties().getProperty("president_percentage_increase"));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public void addStudent(Student student) {
+        students.add(student);
+    }
+
+    public void removeStudent(int index) {
+        if(index >= 0 && index < students.size())
+            students.remove(index);
+    }
+
+    public void removeStudent(Student student) {
+        students.remove(student);
+    }
+
+    public void removeAllStudents() {
+        students.clear();
+    }
+
+    public Student getStudent(int index) {
+        if(index >= 0 && index < students.size())
+            return students.get(index);
+        return null;
```

imo it would be better to throw exception that student is not found instead of returning null.

---

Agree